### ISSUE TITLE
FastTreeDblMP doesn't work in docker

### ISSUE THREAD

**Issue by fanninpm** [nextstrain/docker-base#123]
> I tried running `augur tree --alignment whatever.fasta --method fasttree` and got the following error:

```text
FastTreeDblMP: error while loading shared libraries: libgomp.so.1: cannot open shared object file: No such file or directory
```

_Originally posted by @fanninpm in https://github.com/nextstrain/docker-base/issues/97#issuecomment-1363004029_

____

```text
$ docker image ls --digests nextstrain/base
REPOSITORY          TAG                      DIGEST                                                                    IMAGE ID            CREATED             SIZE
nextstrain/base     build-20221214T204556Z   sha256:e341aadab5681050ddc751218b254b1172276d87516466103db84399fa808d54   84e33405645c        7 days ago          2.41GB
```

**Comment by victorlin** [comment]
@fanninpm thanks for submitting this issue! I can reproduce this with both your provided image digest as well as the latest image. Will look into it.

**Comment by victorlin** [comment]
@fanninpm this should be fixed by #126. The new image is [building](https://github.com/nextstrain/docker-base/actions/runs/3767035254/jobs/6404169396), but you can use the PR's test image if you want to use the fix immediately:

```
nextstrain build --image nextstrain/base:branch-victorlin-fix-fasttree …
```

**Comment by fanninpm** [comment]
I updated to the latest image, and it works like a charm. Thanks!
